eng/pipelines/templates/steps/resolve-build-platforms.yml

```diff
@@ -1,14 +1,14 @@
 parameters:
-  - name: ServiceDirectory
+  - name: PackagePropertiesFolder
     type: string
     default: ''
 
 steps:
     # when we merge pipeline v3, this check will change to examining the targeting string $(TargetingString)
-    # as the generate-pr-diff call + resolution will be present in targeting-string-resolve.yml.
+    # as the generate-pr-diff call + resolution will be present in resolve-package-targeting.yml.
     # until then, we simply check to see if we're targeting storage service directory
     - pwsh: |
-        if ("${{ parameters.ServiceDirectory }}" -eq "storage") {
+        if ("${{ parameters.PackagePropertiesFolder }}".Contains("azure-storage-extension")) {
```

Comment:
Will this be in the package folder path? Or did you expect Contains to check for a file in the folder, as I don't think that is what is happening here.

Reply:
This is just straight up wrong.